accumulator/accumulate_signals_workflow.go

```diff
@@ -0,0 +1,159 @@
+package accumulator
+
+import (
+	"strconv"
+	"time"
+	"go.temporal.io/sdk/workflow"
+)
+
+/**
+ * This sample demonstrates how to accumulate many signals (events) over a time period. 
+ * This sample implements the Accumulator Pattern: collect many meaningful things that
+ *   need to be collected and worked on together, such as all payments for an account, or 
+ *   all account updates by account.
+ * This sample models robots being created throughout the time period, 
+ *   groups them by what color they are, and greets all the robots of a color at the end.
+ * 
+ * A new workflow is created per grouping. Workflows continue as new as needed.
+ * A sample activity at the end is given, and you could add an activity to
+ *   process individual events in the processGreeting() method.
+ * 
+ * Because Temporal Workflows cannot have an unlimited size, Continue As New is used 
+ *   to process more signals that may come in.
+ * You could create as many groupings as desired, as Temporal Workflows scale out to many workflows without limit.
+ * You could vary the time that the workflow waits for other signals, say for a day, or a variable time from first 
+ *   signal with the GetNextTimeout() function.
+ */
+
+// SignalToSignalTimeout is them maximum time between signals
+var SignalToSignalTimeout = 30 * time.Second
```

Comment:
```suggestion
const signalToSignalTimeout = 30 * time.Second
```

Use constants if you can. Also no need to expose probably.

Reply:
nice good changed, thanks.